nservicebus/azure-storage-persistence/configuration.md

```diff
@@ -1,45 +1,21 @@
 ---
 title: Configuration
+component: ASP
 summary: Configuring Azure Storage as persistence
 tags:
  - Azure
  - Cloud
  - Persistence
+reviewed: 2016-11-07
 ---
 
 In NServiceBus.Azure the behavior of the `AzureStoragePersister` can be controlled by working with the appropriate configuration section(s) in the `app.config` or by using the code via the [`IProvideConfiguration` adapter](/nservicebus/hosting/custom-configuration-providers.md). Both approaches to configuring the persister can access the same configuration options.
```

Comment:
@Scooletz you could make that a partial and for < Version 1 say 

"In NServiceBus.Azure the behavior of the `AzureStoragePersister` by using the code via the [`IProvideConfiguration` adapter](/nservicebus/hosting/custom-configuration-providers.md)."

and for others keep the other option.

Any reason why you have both in all versions?